### PR TITLE
Connect2All - Synchronous action

### PR DESCRIPTION
**Pull request by GAC-AJO**
Added Synchronous action to Connect2All connector.
Added Synchronous action to Connect2All on-premises connector.

### CODE REVIEW

File: certified-connectors/Connect2All On-Premises/apiDefinition.swagger.json [hunk 1]
```diff
@@ -2,6 +2,23 @@
   "basePath": "/",
   "consumes": [],
   "definitions": {
+    "Actiontype": {
+      "description": "The Actiontype of the synchronous action that you wish to execute",
+      "title": "",
```
Comment (v-EgorKozhadei) [inline certified-connectors/Connect2All On-Premises/apiDefinition.swagger.json:7]: @GAC-AJO please fill in this property, empty properties are not allowed.

File: certified-connectors/Connect2All On-Premises/apiDefinition.swagger.json [hunk 2]
```diff
@@ -2,6 +2,23 @@
   "basePath": "/",
   "consumes": [],
   "definitions": {
+    "Actiontype": {
+      "description": "The Actiontype of the synchronous action that you wish to execute",
```
Comment (v-EgorKozhadei) [inline certified-connectors/Connect2All On-Premises/apiDefinition.swagger.json:6]: @GAC-AJO please make sure that newely added description and x-ms-summary properties are grammatically correct and are free of typographical errors. Concatenated strings (e.g. actiontype) may cause wrong translation and strings in CaMel case are considered non-translatable.
More information about English strings can be found in [the documentation](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#create-quality-english-language-strings).

File: certified-connectors/Connect2All On-Premises/apiDefinition.swagger.json [hunk 3]
```diff
@@ -2,6 +2,23 @@
...
+        "parameters": {
+          "Company": {
+            "parameter": "Company"
+          },
+          "Url": {
+            "parameter": "Url"
+          }
+        }
+      },
+      "x-ms-summary": "Actiontype"
```
Comment (v-EgorKozhadei) [inline certified-connectors/Connect2All On-Premises/apiDefinition.swagger.json:20]: @GAC-AJO mentioned above. Please change it if there's no specific reason to keep it this way.

File: certified-connectors/Connect2All/apiDefinition.swagger.json [hunk 4]
```diff
@@ -2,6 +2,23 @@
   "basePath": "/",
   "consumes": [],
   "definitions": {
+    "Actiontype": {
+      "description": "The Actiontype of the synchronous action that you wish to execute",
```
Comment (v-EgorKozhadei) [inline certified-connectors/Connect2All/apiDefinition.swagger.json:6]: @GAC-AJO same here. Please fix description, empty title and x-ms-summary.

**Review by v-EgorKozhadei — COMMENTED** [review]
Hello @GAC-AJO,

Please check comments after the review.

Let us know if you have any questions.

File: certified-connectors/Connect2All On-Premises/apiDefinition.swagger.json (hunk 1 above)
Comment (GAC-AJO) [reply certified-connectors/Connect2All On-Premises/apiDefinition.swagger.json]: Changed Actiontype to action type.

File: certified-connectors/Connect2All On-Premises/apiDefinition.swagger.json (hunk 2 above)
Comment (GAC-AJO) [reply certified-connectors/Connect2All On-Premises/apiDefinition.swagger.json]: Changed Actiontype to action type.

File: certified-connectors/Connect2All On-Premises/apiDefinition.swagger.json (hunk 3 above)
Comment (GAC-AJO) [reply certified-connectors/Connect2All On-Premises/apiDefinition.swagger.json]: Changed Actiontype to action type.

File: certified-connectors/Connect2All/apiDefinition.swagger.json (hunk 4 above)
Comment (GAC-AJO) [reply certified-connectors/Connect2All/apiDefinition.swagger.json]: Changed Actiontype to action type.
Added title.

File: certified-connectors/Connect2All On-Premises/apiDefinition.swagger.json (hunk 1 above)
Comment (GAC-AJO) [reply certified-connectors/Connect2All On-Premises/apiDefinition.swagger.json]: Filled title property

**Review by v-EgorKozhadei — APPROVED** [review]

**Comment by v-EgorKozhadei** [comment]
Hello @GAC-AJO, 

I hope you are doing well. 

Congratulations, your pull request is approved and merged. Please [submit the files to ISV Studio](https://docs.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#submit-to-isv-studio) or [update your submission](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-updates) using your latest commit ID, please allow us up to 1-2 weeks to review your submission, and our Engineers will notify you in the “Activity Control” area in ISV Studio. Please [make sure that the icon meets all the requirements](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#design-an-icon-for-your-connector), that it’s size is 230x230, icon background is not white or transparent and it matches the iconBrandColor property in apiProperties.json file. 

After making the submission in ISV Studio, please attach an intro.md file in the activity control section. This file is required and it's different from readme.md. You can [find an example of intro.md file on this page](https://docs.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#submit-to-isv-studio) at step 6. 

Please [Create an environment in the Preview region](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-testing#create-an-environment-in-the-preview-region). You will use this environment later to [test your connector](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-testing#test-your-connector) after Microsoft is done with the functional verification for your connector.  

We expect all tests to be completed within 1-2 weeks. 

If your connector passes all tests, the deployment process begins, and it typically takes up to 3 to 4 weeks to deploy your connector to all of our regions. 

Please let me know if you have any questions or run into any problems while creating or updating your submission. 

Thank you very much for working with us.

**Pull request merged**